airflow/decorators/base.py

```diff
@@ -533,6 +533,9 @@ def expand(self, **kwargs: "Mappable") -> XComArg:
     def expand_kwargs(self, kwargs: XComArg, *, strict: bool = True) -> XComArg:
         ...
 
+    def override(self, **kwargs: Any) -> "_TaskDecorator[FParams, FReturn, OperatorSubclass]":
```

Comment:
```suggestion
    def override(self, **kwargs: Any) -> "Task[FParams, FReturn]":
```

This is the user-facing type, more friendly to see in autocompletion.

Reply:
Excellent. Let's do it.